drivers/include/net/netdev2.h

```diff
@@ -72,6 +72,16 @@ typedef enum {
 } netdev2_event_t;
 
 /**
+ * @brief   Received packet status information for most radios
+ *
+ * May be different for certain radios.
+ */
+struct netdev2_radio_rx_info {
+    uint8_t rssi;       /**< RSSI of a received packet */
+    uint8_t lqi;        /**< LQI of a received packet */
```

Comment:
@OlegHahm how about including the number of retries here? But this might be best targeted in another PR?!